### ISSUE TITLE
public workspaces access w/o account

### ISSUE THREAD

**Issue by aswarren** [PATRIC3/patric3_website#1836]
I've encountered a surprising amount of eagerness around public workspaces.
Currently they are only accessible while logged in.
It would be more useful if this wasn't the case.

**Comment by JoshuaVSherman** [comment]
@dmachi @hyoo I'm assuming that we should close this as designed because we want to have an "Owner" for each workspace, true?

**Comment by JoshuaVSherman** [comment]
@dmachi is this something I can help with, just making any public workspace viewable by anyone, including someone who has not logged into PATRIC?

**Comment by dmachi** [comment]
This used to work, so it is not clear to me why it is failing now.  To determine where it needs to be fixed we’ll need to see how it is breaking.  If a request to view the contents of the folder is going to the backend and then it is returning a permission denied type error then it probably needs to be resolved on the backend workspace service (Bob) If the problem is that the frontend is not making the request at all when not logged in, then we’ll need to fix it on the frontend.  In the latter case it should not be all that difficult and might be worth discussing with Neal (or I can help some too on this one).

**Comment by JoshuaVSherman** [comment]
@nconrad hey, I was able to reproduce this issue, there is no request being sent for display of public workspaces when the user is logged out. Can you suggest a fix?

**Comment by nconrad** [comment]
Yeah, I can add this if you want.   I think we need to allow the WorkspaceManager to be invoked without a token, and then just allow the public workspaces content to be displayed.

**Comment by dmachi** [comment]
I was thinking we already made the workspace backend accept this, but could be remembering wrong

**Comment by nconrad** [comment]
Yeah, I tested, and the backend allows no token.  I thought it didn't for some reason--maybe it was changed at some point.

The only other thing I can think of right now is the "copy" button will need to be disabled if there's no token.  

Do you guys want me to implement it?

**Comment by nconrad** [comment]
This is somewhat related to other sharing tickets I need to address today, so I certainly don't mind.

**Comment by dmachi** [comment]
It’s not clear to me how critical it is, so i can’t comment on it’s priority.

**Comment by mshukla1** [comment]
Not critical, but it would certainly be useful as we are using public workspaces to share 
NIAID funded projects data among other things and now people are using public workspaces 
to polish their own data and include links in the paper. 

If it is something very straightforward and not require too much effort, I say go ahead and do it. 

-Maulik
